web/models.py

```
@@ -3046,3 +3046,85 @@
+    updated_at = models.DateTimeField(auto_now=True)
+    max_students = models.PositiveIntegerField(default=30)
+
+    def __str__(self):
+        return f"{self.name} - {self.teacher.username}"
+
+    class Meta:
+        ordering = ['-created_at']
+
+class VirtualClassroomCustomization(models.Model):
+    """Model for storing virtual classroom customization settings."""
+    classroom = models.OneToOneField(VirtualClassroom, on_delete=models.CASCADE, related_name='customization_settings')
```

Pylint (3.3.7)

[convention] Line too long (119/100)

(C0301)